### PR TITLE
Argument based debug

### PR DESCRIPTION
**Pull request by KenwoodFox**
A proposed approach to making testing and debugging easier by using arguments to set the IP, the code should have a default IP but use any IP you pass as an argument, does this work in a launch file?

### CODE REVIEW

File: src/tidalforce_odom/src/odom.py [hunk 1]
```diff
@@ -127,8 +128,13 @@ def update(self, left, right):
     rospy.init_node("odom")
 
     #ip = rospy.get_param("~ip", "roboRIO-1721-FRC")
+    #ip = rospy.get_param("~ip", "localhost") # For testing
     #ip = rospy.get_param("~ip", "10.17.21.55") # First mDNS almost never works on the feild
-    ip = rospy.get_param("~ip", "localhost") # For testing
+    if (len(sys.argv) > 1): # If there is more than one argument
```
Comment (mikeferguson) [inline src/tidalforce_odom/src/odom.py]: you don't actually need the outer parenthesis.

File: src/tidalforce_odom/src/odom.py [hunk 2]
```diff
@@ -29,6 +29,7 @@
 """
 
 import rospy
+import sys
```
Comment (mikeferguson) [inline src/tidalforce_odom/src/odom.py:32]: for more complex command line arguments - and automatic "help" command, check out argparse.

**Review by mikeferguson — APPROVED** [review]
LGTM

File: src/tidalforce_odom/src/odom.py (hunk 1 above)
Comment (KenwoodFox) [reply src/tidalforce_odom/src/odom.py]: So I should just use `if len(sys.argv) > 1:` right?

**Pull request merged**